server/src/com/cloud/storage/listener/StoragePoolMonitor.java

```diff
@@ -99,12 +99,12 @@ public void processConnect(Host host, StartupCommand cmd, boolean forRebalance)
                     }
 
                     Long hostId = host.getId();
-                    s_logger.debug("Host " + hostId + " connected, sending down storage pool information ...");
+                    s_logger.debug("Host " + hostId + " connected, connecting host to shared pool id " + pool.getId() + " and sending storage pool information ...");
```

Comment:
Please wrap in a `if (s_logger.isDebugEnabled)` to check to avoid unnecessary string concatenation when the `DEBUG` level is disabled.